ui/packages/ui/src/main/webapp/js/views/Module.view.js

```diff
@@ -119,11 +119,16 @@ define([
                 this.model.get('iframeLocation') &&
                 this.model.get('iframeLocation') !== ''
               ) {
-                this.$el.html(
-                  '<iframe src="' +
-                    this.model.get('iframeLocation') +
-                    '"></iframe>'
-                )
+                // only some iframes are brought in as relative
```

Comment:
Since you use this type of method in multiple spots, maybe we should move it to a utility function?